README.md

```diff
@@ -65,16 +65,17 @@ If you are using the STARR-OMOP dataset from Stanford (which uses the OMOP CDM),
 1. Download your STARR-OMOP dataset to `[PATH_TO_SOURCE_OMOP]`.
 2. Convert STARR-OMOP => MEDS using the following:
 ```bash
-femr_stanford_omop_fixer [PATH_TO_SOURCE_OMOP] [PATH_TO_SOURCE_OMOP]
-
 # Convert OMOP => MEDS data format
-meds_etl_omop [PATH_TO_SOURCE_OMOP] [PATH_TO_OUTPUT_MEDS]
+meds_etl_omop [PATH_TO_SOURCE_OMOP] [PATH_TO_OUTPUT_MEDS]_raw
```

Comment:
Likewise, needs a comment about num_shards and num_proc

Reply:
I think this is still TBD pending the other MEDS ETL pull request so could we merge this readme fix first then open a new branch for this change?

Reply:
There are no MEDS ETL pull requests pending so I'm unsure what you mean by "TBD".

But that doesn't matter, feel free to merge this as is as we can add more comments later.